dockerfiles/launcher/launcher_funcs.sh

```diff
@@ -166,6 +166,22 @@ docker_run_with_debug() {
   fi
 }
 
+docker_run_with_volumes_lib() {
+  if has_lib_volume; then
+    docker_run_with_debug -v "${CHE_LIB_VOLUME}":/data/lib "$@"
```

Comment:
Hi @TylerJewell , yes both requirements are needed.  The first step is required to map the named Docker volume to the /data/lib directory (inside the che-server container).  The second requirement is needed to tell any workspace containers that start up to use the same volumes that the che-server container is using, i.e. since che-server has a mapping for /data/lib to the named Docker volume, the workspace will have that same mapping also and therefore be sharing the ws-agent resources.